atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/TracingQueryRunner.java

```diff
@@ -94,17 +99,22 @@ private boolean shouldTraceQuery(Set<TableReference> tableRefs) {
     private void logFailedCall(Set<TableReference> tableRefs) {
         log.warn(
                 "A call to table(s) {} failed with an exception.",
-                tableRefs.stream().map(TableReference::getQualifiedName).collect(Collectors.joining(", ")));
+                tableRefs.stream().map(LoggingArgs::tableRef).collect(Collectors.toList()));
```

Comment:
here and below: let's use `LoggingArgs.tableRefs()`? This one I think generates an arg called `tableRef`, so there'll be multiple with the same name - I can't remember if this is handled nicely